pcdsdevices/slits.py

```diff
@@ -643,3 +643,31 @@ class SimLusiSlits(LusiSlits):
     ywidth = Cpt(FastMotor)
     xcenter = Cpt(FastMotor)
     ycenter = Cpt(FastMotor)
+
+
+class JJSlits(SlitsBase):
+    '''
+    Beckhoff Controlled AT-C8-HV JJ X-Ray Slits.
+    The JJSlits class defines motor components for the specific JJ X-Ray slits
+    model AT-C8-HV. This model of JJSlits holds four in-vacuum motors that set
+    the position and opening of the aperture in two dimensions. For each
+    dimension, one motor defines the aperture opening, and another determines the
+    position of the aperture. This class allows an operator to control and scan
+    the position 'center' and opening 'width' of the aperture. The 'JJSlits'
+    class instantiates four sub-devices: 'JJSlits.xwidth', 'JJSlits.xcenter',
+    'JJSlits.ywidth', 'JJSlits.ycenter'.
```

Comment:
flake8 is still showing a line length violation here, so let's adjust slightly
```suggestion
    dimension, one motor defines the aperture opening, and another determines
    the position of the aperture. This class allows an operator to control
    and scan the position 'center' and opening 'width' of the aperture. The
    'JJSlits' class instantiates four sub-devices: 'JJSlits.xwidth',
    'JJSlits.xcenter', 'JJSlits.ywidth', 'JJSlits.ycenter'.
```